roles/st2chatops/vars/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+supported_hubot_adapters:
+ - slack
+ - hipchat
+ - xmpp
+ - flowdock
+ - yammer
+ - spark
+ - irc
+ - mattermost
```

Comment:
FWIW, `mattermost` adapter should work as well, and we have support for it in hubot-stackstorm, but @armab is right, `matteruser` is the one we pack in `st2chatops`.